services/ui-src/src/components/export/ExportedReportFieldTable.tsx

```diff
@@ -0,0 +1,204 @@
+import { ReactElement } from "react";
```

Comment:
I might not need this component, since it doesn't look like we have a ReportFieldTable - but I figure I'll bring it over and delete it if I can't repurpose it